### DISCUSSION TITLE
Svelte Progress

### DISCUSSION THREAD

**Discussion by fkrauthan** [trivago/prettier-plugin-sort-imports#225] [Q&A]
I was wondering if there is already any information about Svelte support? It is listed as coming soon but I couldn't find any issue to track about actual progress of it? At least currently it doesn't seem to do anything to imports in .svelte files.

**Comment by fkrauthan** [comment]
Just following up here? The feature is marked as coming soon already for a while?

**Comment by ptrxyz** [comment]
I would also be interested in this. I am using the plugin for now in Svelte and it works OKish. Yet sometimes it breaks my imports which then messes up the CI, but it is usable.

Still some info from the maintainers would be nice.
